03/app.js

```diff
@@ -0,0 +1,21 @@
+const buttonElements = document.querySelectorAll('button');
+
+const handleClick = function(e) {
+  console.log('clicked')
+  // let text = 'clicked';
+  // e.target.innerText = text;
+  // zastosowałem Twoją propozycję poniżej (textContent), mogłem też zastosować ją w zad 2, ale jak działa to nie ruszam :)
+  e.target.textContent = 'clicked'
+  e.target.removeEventListener('click', handleClick);
+  
+}
+
+
+if(buttonElements) {
```

Comment:
Tutaj `if` jest zbędny - `.querySelectorAll` zwraca tablicę, a ona zawsze jest prawdą.
Nawet jeśli masz `if([])` to będzie to prawda wiec nie ma co sprawdzać.
Po prostu "pętla" się nie wykona jak nie ma elementów :)